target/linux/ipq806x/patches-5.4/997-device_tree_cmdline.patch

```diff
@@ -8,5 +8,5 @@
 +	if (p != NULL && l > 0)
 +		strlcat(data, p, min_t(int, strlen(data) + (int)l, COMMAND_LINE_SIZE));
  
- 	/*
- 	 * CONFIG_CMDLINE is meant to be a default in case nothing else
+     /* CONFIG_CMDLINE_OVERRIDE is used to fallback to a different
```

Comment:
Fussy comment: If you apply 900-arm-add-cmdline-override.patch after this patch you won't need to modify this one.